src/test/java/pl/sdacademy/pewex/product/mappers/CategoryMapperTest.java

```diff
@@ -0,0 +1,33 @@
+package pl.sdacademy.pewex.product.mappers;
+
+import org.junit.jupiter.api.Test;
+import pl.sdacademy.pewex.product.db.CategoryEntity;
+import pl.sdacademy.pewex.product.model.Category;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CategoryMapperTest {
+
+    @Test
+    void shouldMapEntityToCategory(){
```

Comment:
Super mamy pierwszy test!!

To teraz proszę tutaj dodać drugą metodę gdzie do mappera trafi null i szybko wyjdzie, że mamy NPE

Reply:
dodano drugą metodę